core/variables.js

```diff
@@ -417,3 +418,18 @@ Blockly.Variables.promptName = function(promptText, defaultText, callback) {
     callback(newVar);
   });
 };
+
+/**
+ * Generate XML string for variable field.
+ * @param {!Blockly.VariableModel} variableModel The variable model to generate
+ *     an XML string from.
+ * @return {string} The generated XML.
+ * @private
+ */
+Blockly.Variables.generateVariableFieldXml_ = function(variableModel) {
```

Comment:
This function isn't applicable to your code since you use goog.dom.createDom() and manually set attributes instead. So you can get rid of it now or keep it in case we use it in the future.